bin/deploy-site

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+const ghPages = require('gh-pages');
+const path = require('path');
+const fs = require("fs");
+const childProcess = require('child_process');
+
+const packageJson = path.join(process.cwd(), 'package.json');
+
+if (!fs.existsSync(packageJson)) {
+    console.error('Please run in the root directory of the package');
+    process.exit(1);
+}
+
+const scripts = JSON.parse(fs.readFileSync(packageJson, 'utf-8')).scripts
```

Comment:
Missing semicolon at EOL.

Could be simplified a bit with [findup-sync](https://www.npmjs.com/package/findup-sync):

```js
const findUpSync = require('findup-sync');
const packageJson = findUpSync('package.json');
if (!packageJson) {
    console.error('Please run in the root directory of the package');
    process.exit(1);
}
const scripts = require(packageJson).scripts;
```

... That also wouldn't require the cwd to be the root of the repository. Especially not if you go:

```js
childProcess.execSync('npm run generate-site', {cwd: path.dirname(packageJson)})
```

Reply:
Nice - I'll try that.

Reply:
Mhh the gh-pages also needs to run from the root. I think I'll just leave it simple as it is.